### ISSUE TITLE
Include website title in nav bar

### ISSUE THREAD

**Issue by Toflar** [contao/core-bundle#1121]
We use the `websiteTitle` to output the version of a platform quite often, like so:

```yaml
contao:
    localconfig:
        websiteTitle: "Website title (v %platform_version%)"
```

On every update we dynamically adjust the `platform_version` parameter from git tags but that's a detail 😄 

Would you accept displaying the website title there by default so it would look something like this:

![bildschirmfoto 2017-10-02 um 17 46 12](https://user-images.githubusercontent.com/481937/31086480-34dbb94c-a79a-11e7-9e8f-4e74f5140db1.png)

If you think that's a bad idea to output the title there, would you adjust `be_main` so that it gets a block here so I don't have to override all of it?

```html
        <div class="version">
          <?= $this->version ?><br><?= $this->learnMore ?>
        </div>
```

**Comment by Toflar** [comment]
Because right now the website title is only visible on the welcome screen where a user almost never really is. It also eases support. If I ask "which version are you using?", the usual answer is "where can I see that?" and I think that would be gone then 😄

**Comment by fritzmg** [comment]
> Because right now the website title is only visible on the welcome screen …

And the site structure ;)

**Comment by Toflar** [comment]
True 😄

**Comment by leofeyer** [comment]
Of course we can add blocks, which would also cover #1122. What blocks do we need?

**Comment by Toflar** [comment]
Uff...proposal:

- head
    - title
    - meta
    - stylesheets
    - scripts
- header
- container
    - left
         - version
    - main
        - main_headline
       - main_content
- mootools

maybe?

**Comment by aschempp** [comment]
I don't think blocks make sense, because a backend template cannot be extended. We can't change the name of the template 😉

**Comment by leofeyer** [comment]
You don't have to change the template name to extend it.

**Comment by aschempp** [comment]
As far as I know you do? You cannot extend a template with the same name of the current template, that would be recursive? Or how is the templateloader supposed to know you want to extend the bundle one and not the root?

**Comment by fritzmg** [comment]
It works through magic from @ausi afaik ;)

**Comment by leofeyer** [comment]
Exactly.

**Comment by ausi** [comment]
@aschempp see contao/core#6939

**Comment by leofeyer** [comment]
Implemented in bb8765bda202fad4abf8a0f2ed31b7e65104b88f with a little less blocks than suggested, because blocks do not make sense if the content is already stored in a variable (e.g. `$this->headline`). Let me know if you need more blocks though.
